R-packages/evalcast/R/empty_.R

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
What is the convention for the trailing underscore in the file name here?  If this is supposed to be a file that isn't user facing, the Python convention is to lead with the underscore, which I find much more readable.